scripts/decode_path_json.py

```
@@ -0,0 +1,288 @@
+    invariants = {}
+    invariant_bits = lookup_tables["invariant_bits"]
+    
+    for bit_str, name in invariant_bits.items():
+        bit_pos = int(bit_str)
+        passed = bool(bitfield & (1 << bit_pos))
+        invariants[name] = passed
+    
+    return invariants
+
+
+def reconstruct_transaction(compressed_data: Dict, lookup_tables: Dict, addresses: List[str]) -> Tuple[TransactionRoundResults, TransactionBudget]:
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)